receiver/opencensusreceiver/octrace/opencensus.go

```diff
@@ -145,18 +154,14 @@ func (ocr *Receiver) sendToNextConsumer(longLivedRPCCtx context.Context, traceda
 		receiverTransport,
 		obsreport.WithLongLivedCtx())
 
-	if c, ok := client.FromGRPC(ctx); ok {
-		ctx = client.NewContext(ctx, c)
-	}
-
 	var err error
-	numSpans := 0
+	numSpans := len(tracedata.Spans)
```

Comment:
This can cause sigfault.

Reply:
Not in practice since the only caller always allocate it right before the call, that said this parameter as pointer seems like an early optimization, I will change it to a value type like `ConsumeTraceData` function.